tutorials/appgyver-subscribe-service/appgyver-subscribe-service.md

```diff
@@ -10,7 +10,11 @@ author_profile: https://github.com/akseliv
 ---
 
 ## Prerequisites
- - Access to an SAP BTP account in EU10 with Low-Code / No-Code entitlements
+ - Access to an SAP BTP account in EU10 with paid Low-Code / No-Code entitlements.
+
+ Note: This is a paid service, with further details available at [SAP Discovery Center]  (https://discovery-center.cloud.sap/serviceCatalog/sap-appgyver?region=europe(frankfurt)&service_plan=standard&tab=feature)
```

Comment:
Please adjust formatting and don't start a note with the label "Note".
https://jam4.sapjam.com/wiki/show/MozDWaPczznL8NKeGEk0y9#mcetoc_1dtqfqchq21